test/JDBC/input/ownership/babelfish_authid_login_ext-vu-verify.sql

```diff
@@ -9,4 +9,5 @@ GO
 
 SELECT rolname FROM sys.babelfish_authid_login_ext 
 WHERE rolname LIKE 'babelfish_authid_login_ext_vu_prepare_login%'
+ORDER BY rolname
```

Comment:
Just to be sure, should we add `order by` clause in the first line of this file?

Reply:
That's not needed because the view already includes ORDER BY when it's created